### PR TITLE
PSD-280: add data call with Graphql flss query to filter-alpha page

### PR DESCRIPTION
**Pull request by go-bears**
Jira ticket:
https://kiva.atlassian.net/browse/PSD-280

What does it do:
adds flss query to filter-alpha page to provide access to live loans and data for component presentation. 

How to test:
download the branch and run the dev server
when the `filter-alpha` page renders open the developer tools console and look for `console.log` outputs for loans from the US and the totalCount number of available loans.

### CODE REVIEW

File: src/graphql/query/flssLoansQuery.graphql [hunk 1]
```diff
@@ -1,10 +1,10 @@
 #import '../fragments/loanCardFields.graphql'
 
 query flssLoanData(
-  $filterObject: [FundraisingLoanSearchFilterInput!] = { countryIsoCode: {any: ["US"]}}
+  $filterObject: [FundraisingLoanSearchFilterInput!]
   $imgDefaultSize: String = "w480h360"
   $imgRetinaSize: String = "w960h720"
-  $limit: Int = 20
+  $limit: Int = 50
```
Comment (mcstover) [inline src/graphql/query/flssLoansQuery.graphql:7]: Please lower this default. This file will now become shared so we shouldn't be asking for 50 loans at a time by default.

**Review by mcstover — APPROVED** [review]
Beyond the default fetch limit, this is a good foundation to build on!

**Review by ryan-ludwig — APPROVED** [review]

**Comment by go-bears** [comment]
Thanks for 👍  @mcstover @ryan-ludwig ! I reset the limit to 20 but moved where that gets controlled to the vue page.

File: src/graphql/query/flssLoansQuery.graphql [hunk 2]
```diff
@@ -1,10 +1,10 @@
 #import '../fragments/loanCardFields.graphql'
 
 query flssLoanData(
-  $filterObject: [FundraisingLoanSearchFilterInput!] = { countryIsoCode: {any: ["US"]}}
+  $filterObject: [FundraisingLoanSearchFilterInput!]
   $imgDefaultSize: String = "w480h360"
   $imgRetinaSize: String = "w960h720"
-  $limit: Int = 20
+  $limit: Int = 0
```
Comment (meganmcmillan) [inline src/graphql/query/flssLoansQuery.graphql:7]: I could see this biting someone in the future if they don't set a limit in their component - "why is my query returning 0 loans???". We probably want to keep this default limit as 10-20 here, which can then be overridden in the component.

**Review by meganmcmillan — COMMENTED** [review]
One comment. Overall this looks good!

File: src/graphql/query/flssLoansQuery.graphql (hunk 2 above)
Comment (go-bears) [reply src/graphql/query/flssLoansQuery.graphql]: thanks for 👀 !

**Pull request merged**